File: VB/WindowsApplication1/Component/LookAndFeelSettings.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports DevExpress.LookAndFeel

Namespace WindowsApplication1
	<Serializable> _
	Public Class LookAndFeelSettings
		Public SkinName As String
		Public Style As LookAndFeelStyle
		Public UseWindowsXPTheme As Boolean

		Public Sub New()

		End Sub
	End Class
End Namespace

File: CS/WindowsApplication1/Component/LookAndFeelSettings.cs
using System;
using System.Collections.Generic;
using DevExpress.LookAndFeel;

namespace WindowsApplication1
{
    [Serializable]
    public class LookAndFeelSettings
    {
        public string SkinName;
        public LookAndFeelStyle Style;
        public bool UseWindowsXPTheme;

        public LookAndFeelSettings()
        {

        }
    }
}
